app/src/androidTest/java/ch/hikemate/app/ui/map/MapScreenTest.kt

```diff
@@ -3,17 +3,22 @@ package ch.hikemate.app.ui.map
 import androidx.compose.ui.test.assertIsDisplayed
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
+import ch.hikemate.app.ui.navigation.NavigationActions
+import ch.hikemate.app.ui.navigation.TEST_TAG_SIDEBAR_BUTTON
 import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.mockito.Mockito.mock
 
 class MapScreenTest : TestCase() {
   @get:Rule val composeTestRule = createComposeRule()
+  lateinit var navigationActions: NavigationActions
 
   @Before
   fun setUp() {
-    composeTestRule.setContent { MapScreen() }
+    navigationActions = mock(NavigationActions::class.java)
```

Comment:
Question: might be a dumb question, but does it make sense to mock the `navigationActions`? Can the test not use a "real" navController to initialize a "real" navigationActions? I'm thinking about reducing the effort necessary for mocking to a minimum.

Reply:
Since I am not really testing the functionalities of the `NavigationActions` here I'd say it does. But both are correct.
I am mocking it only to get the parameter for the code to compile.

Reply:
You could revert it like this: 
`navHostController = NavHostController(InstrumentationRegistry.getInstrumentation().targetContext)`
    `navigationActions = NavigationActions(navHostController)`

Reply:
Yes, thanks!